weave/flow/prompt/common.py

```diff
@@ -0,0 +1,19 @@
+from typing import Any, Literal
+
+Templating = Literal["weave", "none"]
```

Comment:
What are these templating formats? how is `weave` different than `fstrings?`

Reply:
This is something I'm still thinking about. "none" turns off the template parsing, which I think we probably want to do for assistant responses by default.

My fuzzy idea for our parameters is that we would generate a form UI from them in the prompt playground. So compared to f-strings where the controls are for things like numeric precision it would be more like `type:slider min:0 max:100`.